frontend/app/src/components/status/update/AssetConflictDialog.vue

```diff
@@ -95,13 +95,28 @@ function isDiff(conflict: AssetUpdateConflictResult, field: AssetKey) {
 
 const remaining = computed(() => {
   const resolved = get(resolutionLength);
-  return get(conflicts).length - resolved;
+  return get(conflicts)
+    .map(({ identifier }) => identifier)
+    .filter(uniqueStrings).length - resolved;
```

Comment:
can we do 
`return uniqueObjects(get(conflicts), item => item.identifier).length - resolved`
to not do 2 loops here